lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -908,6 +890,29 @@ def scontrol(*args) -> str:
         return _call("scontrol", *args).stdout
 
 
+class SackdManager(_SlurmManagerBase):
+    """Manager for the Sackd service."""
```

Comment:
Does SACKD_CONFIG_SERVER need documented here as SLURMD_CONFIG_SERVER is for SlurmdManager?

Reply:
It should be documented! I'm going to clean up the docstrings while the work on enabling the `apt` charm library to handle deb822 formatted sources.

Reply:
Fixed in latest batch of commits. Took some time to improve the docstrings because some were out of date and providing inaccurate information.